lib/repositories/i_repository.dart

```diff
@@ -0,0 +1,3 @@
+abstract class IRepository {
+  Future<List<dynamic>> getData();
```

Comment:
Use a generic type `<T>` for the interface. Avoid using `dynamic`

Reply:
Done